dnf/repo.py

```diff
@@ -694,6 +697,7 @@ def _handle_new_remote(self, destdir, mirror_setup=True):
         mirrorlist = self.metalink or self.mirrorlist
         if mirrorlist:
             h.hmfcb = self._md_pload._mirror_failure_cb
+            h.starttransfercb = self._md_pload._start_transfer_cb
```

Comment:
Why in section `if mirrorlist`? Probably you want to set `h.starttransfercb` for repositories with baseurl too.

Reply:
done